documentation/src/main/asciidoc/topics/json/persistence_sql_query.json

```
@@ -6,24 +6,18 @@
         "shared": "true",
         "key-columns": "isbn",
         "queries": {
-          "select-single": "SELECT isbn, title FROM books WHERE isbn = :isbn",
-          "select-all": "SELECT isbn, title FROM books",
+          "select-single": "SELECT books.id, books.isbn, title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id WHERE books.id=:id",
+          "select-all": "SELECT books.id, books.isbn, title, authors.name, authors.country FROM books JOIN authors ON books.author_id = authors.id",
           "delete-single": "DELETE FROM books WHERE isbn = :key",
```

Same thing this delete is no longer valid as the key doesn't match.
```suggestion
          "delete-single": "DELETE FROM books WHERE id = :id",
```